heudiconv/cli/run.py

```diff
@@ -112,6 +128,19 @@ def main(argv=None):
         random.seed(args.random_seed)
         import numpy
         numpy.random.seed(args.random_seed)
+    # Ensure only supported bids options are passed
+    allowed_options = [option for option, _ in BIDS_OPTIONS]
```

Comment:
argparse's `choices` will check if each input is valid, so this isn't needed

Reply:
Done.